### ISSUE TITLE
Stop playback

### ISSUE THREAD

**Issue by drizzt09** [dmhacker/alexa-youtube-skill#45]
I have successfully set it up and its finding and playing the songs requested.
"Alexa, Ask Youtube to play artist title"

However during playback if i say any of these:
Alexa, stop
Alexa, ask Youtube to stop
Alexa, ask Youtube to pause

She always responds with:
"You are not playing anything currently"

Also on a side note... If i want o switch to my own Heroku its only the URL i have to change in the AWS Lambda index.js?
// Set Heroku URL
var heroku = process.env.HEROKU_APP_URL || "https://**********.herokuapp.com";

Nothing else requires changing?

**Comment by Manav-Seth** [comment]
hmm, Alexa, ask Youtube to stop works for me.

**Comment by Manav-Seth** [comment]
But I am puzzled why Alexa stop does not work? It most likely has to do with shouldEndSession flag not being set?

**Comment by drizzt09** [comment]
So are there logs I can capture to help find out why

**Comment by Manav-Seth** [comment]
@drizzt09 let me know how to capture logs, and I will be happy to provide them

**Comment by dmhacker** [comment]
Hmm, from my testing, resume/pause works as intended. Stop doesn't work; you need to say cancel instead for unknown reasons.

As for changing the Heroku URL, just set the HEROKU_APP_URL environment variable in your Lambda server.

**Comment by jarobar435** [comment]
I just done your Setup Process tutorial recently and also got the problem with pause/stop/resume commands.
I am able to use commands "Alexa pause" and then "Alexa resume" for once each track. Then everything goes wrong ("You are not playing anything currently"...) - I have no control over the playback anymore - can't pause or stop it anyway. So literally that means I have to listen to it till very end or unplug my Echo Dot.
So it seems to be the problem with "resume" not setting some kind of flag/boolean.
If you need any detailed feedback - I'm at your service.

**Comment by dmhacker** [comment]
Yeah, that functionality is a bit buggy; I'm still working out some of the kinks. Try saying "cancel" rather than "pause" or "stop" to completely shutdown the skill if things get out of hand. Let me know if that helps.
